### PR TITLE
55 refactor posts part of the documentation

### PR DESCRIPTION
**Pull request by Antonyjin**
Refactoring the posts part of the documentation.

Splitting main points to make it more clear.

- Adding information (proto + arguments)
- Adding examples
- Adding forget usage

### CODE REVIEW

File: docs/libraries/python-sdk/Posts/create.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+- Events
+- Blog posts
+- Comments
+- and even be used to emulate a [public database](https://github.com/aleph-im/active-record-sdk)
+
+!!! note
+
+    More information on posts and messages in general can be found in the
+    [protocol section](../../protocol/messages.md).
```
Comment (hoh) [inline docs/libraries/python-sdk/Posts/create.md:11]:
```suggestion
    [protocol section](../../../protocol/messages.md).
```

File: docs/libraries/python-sdk/Posts/create.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+Posts are unique data entries. For example:
```
Comment (hoh) [inline docs/libraries/python-sdk/Posts/create.md:1]: Reference directly the page about posts `protocol/object_types/posts/`

File: docs/libraries/python-sdk/Posts/query.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+### Example
+
+```python
+from aleph.sdk.client import AlephHttpClient
+
+async def main():
+    async with AlephHttpClient() as client:
+        message = await client.get_message(
+            "142f99fbbe63befc868d16a4e389395cd1abc812b0eecaba72e08b8709caafd6"
+        )
```
Comment (hoh) [inline docs/libraries/python-sdk/Posts/query.md:103]:
```suggestion
            "142f99fbbe63befc868d16a4e389395cd1abc812b0eecaba72e08b8709caafd6",
            message_type = PostMessage,
        )
```

File: docs/libraries/python-sdk/Posts/query.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        return message
+
+import asyncio
+message = asyncio.run(main())
+print(message.content)
```
Comment (hoh) [inline docs/libraries/python-sdk/Posts/query.md:108]:
```suggestion
from pprint import pprit
pprint(message.content.dict())
```

**Review by MHHukiewitz — COMMENTED** [review]
Thanks, two questions:
- How did you create these tables for function arguments? Did you use a generative tool for that?
- Error seems to be empty, what should this page be about?

**Comment by Antonyjin** [comment]
I created them manually using the data tables in mkdocs : 
https://squidfunk.github.io/mkdocs-material/reference/data-tables/

For the error page, I thought I'd done it. It was a matter of calling get_messages() and get_messages_error() to find out the status of a request and, in the event of a problem, to have logs about it.

**Comment by Antonyjin** [comment]
As a reminder, this PR was originally intended to re-document the Post method.

I've just completed the rebasing with the main branch and written the explanation of the error page.

What do you think of this version?

**Comment by MHHukiewitz** [comment]
The Error section looks good, but it would be even greater if we could list some of the common errors. For that, you could look into the pyaleph code to find a list of error codes.

About the size, I don't mind that much. I'll try to review the pages that were changed in the preview and if they read well, I approve.

**Comment by Antonyjin** [comment]
This is a great idea, I'm going to do that!

**Comment by Antonyjin** [comment]
I've created a table with all the error codes instead of just a few.
I find it more complete and the display is very nice.

What do you think?

**Review by MHHukiewitz — APPROVED** [review]
LGTM :+1:

**Pull request merged**